### ISSUE TITLE
[Feature Request] Uninstall/Install multiple sources?

### ISSUE THREAD

**Issue by KaiserBh** [Jays2Kings/tachiyomiJ2K#944]
### Why/User Benefit/User Problem
Having to install multiple sources can be annoying, well the main reason is I wanted to install like 15+ sources (I realised I don't need that much now and want to remove them) but for instance a user like me might have installed a lot of useless sources and that's not needed and want it to be removed without having to go to individual sources would be great and less time consuming.

 I don't know if this is possible or have been request previously but I did my best to search it up and couldn't find anything on the issues.

I kinda installed like almost all the sources and regret it and now I could go there and uninstall it manually it just takes time, if there is an automated version of it would be appreciated again if this is possible. 

### What/Requirements
(explain how this feature would behave)
Basically we let the users pick how many sources they want to install or uninstall and then the app will remove the sources or install it without manually going through each one and install/uninstall them.

**Comment by CarlosEsco** [comment]
This is a limitation of android without root.  It's possible in the future Android 12 devices will be able to do this without manual intervention
